inference-engine/tests/functional/plugin/shared/include/behavior/vpu_scale_test.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
I think that plugin-specific tests should not be placed in the shared directory. It might be better to put them in `unit/vpu/middle end_tests/passed_tests`